app/agents_v2/primary_agent/prompts/response_formatter.py

```
@@ -6,11 +6,12 @@
 quality standards defined in the enhancement specification.
 """
 
-from typing import Dict, List, Optional, Tuple
+from typing import Dict, List, Optional, Tuple, Union
```

Ruff (0.12.2)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)